lib/gettext_i18n_rails/tasks.rb

```diff
@@ -101,6 +101,9 @@ def gettext_xgettext_options
 
     language_path = File.join(locale_path, language)
     mkdir_p(language_path)
-    ruby($0, "gettext:find")
+    b = false
+    scope = _.scope.reject{|x| b || x == 'gettext' ? b = true : false}.join(':')
```

Comment:
otherwise this + some comment on why we are doing this crazyness :)

```
prefix = _.scope.slice(0, _.scope.index('gettext'))
scope = (prefix + ['gettext:find']).join(':')
ruby($0, scope)
```